doc/manual/getting_started_core.rst

```diff
@@ -151,16 +151,29 @@ It is often necessary that several pulses overlap one another. This can be expre
 
 Try the following code and observe the generated pulses on a 2-channel oscilloscope or logic analyzer: ::
 
-    for i in range(1000000):
-        with parallel:
-            self.ttl0.pulse(2*us)
-            self.ttl1.pulse(4*us)
-        delay(4*us)
+    from artiq.experiment import *
+
+
+    class Tutorial(EnvExperiment):
+        def build(self):
+            self.setattr_device("core")
+            self.setattr_device("ttl0")
+            self.setattr_device("ttl1")
+
+        @kernel
+        def run(self):
+            self.core.reset()
+            self.ttl0.output()
```

Comment:
This ``output()`` call seems to be specific to your setup? As it's not present in the example above, and only ``ttl0`` is affected; and on Sinara TTL cards it will not do anything either.

Reply:
I think it's just there to ensure that ttl0 is in output mode. For consistency, there should be the same statement for ttl1 or we should drop both of them. I don't have a particularly strong opinion either way.